plugin/completion.py

```diff
@@ -69,7 +69,7 @@ def format_completion(
         annotation = lsp_label_description or lsp_detail
     elif lsp_label.startswith(lsp_filter_text):
         trigger = lsp_label
-        annotation = lsp_detail
+        annotation = lsp_label_description or lsp_detail
         if lsp_label_detail:
             details.append(html.escape(lsp_label + lsp_label_detail))
         if lsp_label_description:
```

Comment:
If `lsp_label_description` is used here then I would probably not add it to `details` since it would just be a duplicated data then.

Reply:
To not loose any information, we should put `lsp_detail` in `details` if `lsp_label_description` is used and vice-versa.